redi-connect-front/src/components/organisms/RediHero.scss

```diff
@@ -0,0 +1,37 @@
+@import "~bulma/sass/utilities/_all";
+@import '../../styles/variables.scss';
+
+.hero {
+  &__column {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+
+    &--heading.title {
```

Comment:
We should try to use the design system and not define special line-heigts. 
This will just lead to repeating your self. If the lineheight is not correct check your options from Bulma. 
If the configs are not good update the options. This changes should be possible with Bulma maybe just the variables are not correct...